### PR TITLE
docs: Add some doc to contribute to the project

### PR DESCRIPTION
**Pull request by GuillaumeAmat**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+```
+$ git checkout develop
+$ npm version patch -m "release: %s"
+$ npm publish
+$ git checkout master
+$ git merge develop
+$ git push origin master
+```
+
+`npm version` tests the code and build it. Then it upgrades the package version number according to the used keyword (patch, minor or major) and commit the modifications in Git (with a proper version tag). Finally, it pushes it to repository with the tag.
```
Comment (bleucitron) [inline CONTRIBUTING.md]: "tests the code and build**s** it"
"and commit**s**"

:p

**Review by bleucitron — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (bleucitron) [reply CONTRIBUTING.md]: I'm a bitch, I know

Comment (GuillaumeAmat) [reply CONTRIBUTING.md]: ^^ It is fixed

**Review by bleucitron — APPROVED** [review]

**Comment by bleucitron** [comment]
Ok !
I see that you `git pull develop`, leading to a 'Merge' commit.
In the future we should avoid it, and `rebase` instead, to keep the repo clean.

**Pull request merged**